pypesto/model_selection/constants.py

```diff
@@ -0,0 +1,23 @@
+#YAML_FILENAME_COLUMN = "SBML"
+#MODEL_NAME_COLUMN = "modelId"
```

Comment:
```suggestion
# YAML_FILENAME_COLUMN = "SBML"
# MODEL_NAME_COLUMN = "modelId"
```

Reply:
ore remove ...

Reply:
should be removed since it is taken care of by lines 4,6,7